src/js/components/ServicePlanBlocks.js

```diff
@@ -0,0 +1,193 @@
+import classNames from 'classnames';
```

Comment:
you have a struct named `ServicePlanBlocks` and this component is named `ServicePlanBlocks`. can we try to name it something different?